src/ports/server/component.ts

```diff
@@ -134,8 +173,7 @@ export async function createServerComponent({
         code: request.code,
         method: request.method,
         params: request.params,
-        sender: request.sender,
-        chainId: request.chainId
+        sender: request.sender
```

Comment:
Should the sender be lowercased the same way as it's done when storing it?

Reply:
it is already lowercased when stored, and the one here is the same one so it will be already lowercased